dev/tests/static/framework/Magento/Sniffs/Annotation/MethodArgumentsSniff.php

```diff
@@ -681,4 +681,17 @@ private function noneParamsAligned(array $argumentPositions, array $commentPosit
 
         return $flag;
     }
+
+    /**
+     * Get argument name starting position in annotation string.
+     *
+     * @param string $paramContent
+     * @param array $paramDefinition
+     * @return int
+     */
+    private function getArgumentPosition(string $paramContent, array $paramDefinition): int
```

Comment:
I am not 100% this can happen, but looking at the code, seems that params are retrieved from annotations in `\Magento\Sniffs\Annotation\MethodArgumentsSniff::process`. Did you test what happens if a param is missing?
Can `strpos` give a `false` result in this condition? If this happens you could get a strict type validation error since you specified `int` as return type.

Thank you.

Thank you.

Reply:
@phoenix128 thank you for review. You're right about this concern. I've updated the code.

Reply:
@phoenix128 thank you for review. You're right about this concern. I've updated the code.